### PR TITLE
Adding MongoDb scenarios to graphs

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

File: src/Benchmarks/Data/ApplicationDbSeeder.cs [hunk 1]
```diff
@@ -1,97 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (mikeharder) [inline src/Benchmarks/Data/ApplicationDbSeeder.cs]: Why was this file removed?

Comment (sebastienros) [reply src/Benchmarks/Data/ApplicationDbSeeder.cs]: All the docker images we have for the databases are using custom scripts which are already seeded. And because it was showing error messages I decided to remove it.
